jaeger-core/src/main/java/io/jaegertracing/reporters/Reporter.java

```diff
@@ -14,7 +14,7 @@
 
 package io.jaegertracing.reporters;
 
-import io.jaegertracing.Span;
+import io.opentracing.Span;
```

Comment:
I don't think this is a good direction. The only OpenTracing facade this library provides is the Tracer API. Reporter and everything else are internal components that should only work with internal span type, and the Tracer is the one responsible for downcasting from OT span to our span. In particular, for reporting we don't even need downcasting, because `JaegerSpan.finish()` calls `tracer.report(this)`, so reporter gets the internal span directly.